in_text/text_data_release.yml

```diff
@@ -104,58 +134,66 @@ entities:
     attributes:
     - 
       attr-label: time
```

Comment:
Consider checking and then adding the actual ranges for `time`, `ShortWave`, `LongWave`, `AirTemp`, `WindSpeed`, and `Rain`.